libraries/lib-theme/AllThemeResources.h

```diff
@@ -374,3 +386,7 @@ from there.  Audacity will look for a file called "Pause.png".
 
    DEFINE_COLOUR( clrClipNameText,          wxColour(0, 0, 0),       wxT("ClipNameText") );
    DEFINE_COLOUR( clrClipNameTextSelection, wxColour(183, 202, 226), wxT("ClipNameTextSelection") );
+
+   DEFINE_COLOUR( clrDropHintHighlight, wxColour( 71, 140, 247 ), wxT("DropHintHighlight") );
+   DEFINE_COLOUR( clrEffectListItemBackground, wxColour( 60, 60, 60 ), wxT("EffectListItemBackground"));
+   DEFINE_COLOUR( clrEffectListItemBorder, wxColour( 204, 204, 204 ), wxT("EffectListItemBorder"));
```

Comment:
This changes the format for the monolithic "sprite sheet" for dropping in a custom theme.  After merging this we must remember to give @LWinterberg the updated sheet to put in the Wiki page.  If more changes to this file are expected during this release cycle, then that can wait until they are all done.